src/theme/BoxLabel.ts

```diff
@@ -29,33 +30,33 @@ export const BoxLabel: ComponentStyleConfig = {
 
       return {
         bg: {
-          primary: "brand.50",
-          secondary: "gray.50",
+          primary: mode("brand.50", "whiteAlpha.200")(props),
+          secondary: mode("gray.50", "whiteAlpha.100")(props),
           magic: colors.gradient[3],
-          success: "green.50",
-          warning: "yellow.50",
-          error: "red.50",
+          success: mode("green.100", "green.200"),
+          warning: mode("yellow.100", "yellow.200"),
+          error: mode("red.100", "red.200"),
```

Comment:
I think we should pass `prop` there:

```suggestion
          success: mode("green.100", "green.200")(props),
          warning: mode("yellow.100", "yellow.200")(props),
          error: mode("red.100", "red.200")(props),
```